config/locales/devise.en.yml

```diff
@@ -46,7 +46,7 @@ en:
       updated: "Your account has been updated successfully."
       updated_but_not_signed_in: "Your account has been updated successfully, but since your password was changed, you need to sign in again."
     sessions:
-      signed_in: "Signed in successfully."
+      signed_in: ""
```

Comment:
Doesn't this just make the message blank?  I think we need to prevent the message from being sent.